kubernetes/Chart.lock

```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: postgresql
   repository: https://charts.bitnami.com/bitnami/
-  version: 12.11.1
-digest: sha256:3c10008175c4f5c1cec38782f5a7316154b89074c77ebbd9bcc4be4f5ff21122
-generated: "2023-09-14T22:40:43.171275362Z"
+  version: 14.3.1
```

Comment:
The major version bump here might cause issues for existing deployments

Reply:
Hmmm.. That's true, but we also bumped major helm version.
What should we do? Leave very old helm chart?

Reply:
Perhaps just leave it as `version: 12.11.1`, but add a note saying that a newer Postgres version should be used if it's a fresh deployment?

Reply:
I also bumped the image from postgres:13 to postgres:16

Reply:
Should probably change that as well if getting rid of the version bump in the Postgres chart

Reply:
True, but only if we want to get rid of it, I'll wait for more comments.
I bumped helm version to new major as it's not strict-compatible with older version - `securityPolicy` is the next changed thing.

Reply:
OK with the postgresql version bump.

We do not maintain an helm repository yet so people should look at the new features before applying the new helm chart.

Now that I think about it, I think we should extract the helm chart from the repository and do it like with searxng in a separate repo: https://github.com/searxng/searxng-helm-chart